entries/m/mailjet.com.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Mailjet": {
+    "domain": "mailjet.com",
+    "img": "mailjet.com.png",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://documentation.mailjet.com/hc/en-us/articles/10553745110043-Two-factor-authentication-2FA",
```

Comment:
```suggestion
    "documentation": "https://documentation.mailjet.com/hc/articles/10553745110043",
```

Reply:
Thank you! I have fixed this problem by myself! ❤️